.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Animal Farm NodeJS CI
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Anything above Ubuntu 20.04.5 LTS broke my bot. Is latest above 20.04.5?

Reply:
That is strange and unexpected...

Reply:
Naybe we set a version to not have to worry about future incompatibilities?